setup.py

```diff
@@ -21,8 +21,8 @@
         'requests',
         'six',
         'shub>=0.2.5',
-        'docker-py',
     ],
+    extras_require={'docker-py': ['docker-py>=1.10.0']},
```

Comment:
Sorry, I was beat out of reason when seeing a check for docker-py [here](https://github.com/scrapinghub/shub-image/blob/master/shub_image/tool.py#L27).
We don't need this check then, do we?